client/a8c-for-agencies/sections/sites/controller.tsx

```diff
@@ -1,14 +1,13 @@
 import { Context, type Callback } from '@automattic/calypso-router';
 import { setAllSitesSelected } from 'calypso/state/ui/actions';
 import SitesSidebar from '../../components/sidebar-menu/sites';
-import { A4A_SITES_DASHBOARD_DEFAULT_FEATURE } from './constants';
 import SitesDashboard from './sites-dashboard';
 import { SitesDashboardProvider } from './sites-dashboard-provider';
 
 function configureSitesContext( isFavorites: boolean, context: Context ) {
 	const category = context.params.category;
 	const siteUrl = context.params.siteUrl;
-	const siteFeature = context.params.feature || A4A_SITES_DASHBOARD_DEFAULT_FEATURE;
```

Comment:
The `A4A_SITES_DASHBOARD_DEFAULT_FEATURE` is a dynamic param.

Reply:
I don't quite get it. Where will it be set up?

Reply:
Nowhere... it is not necessary to specify globally it. It's configured in its context, in the `JetpackPreviewPane` component (see comments below)